app/helpers/application_helper.rb

```diff
@@ -13,4 +13,12 @@ def document_collection_title(value:, **)
   def document_leaflet_map(document:, **)
     render_document_partial(document, 'show_leaflet_map_wrapper')
   end
+
+  ##
+  # @param [String] manifest
+  def iiif_drag_n_drop(manifest)
+    link_to mirador_index_path(manifest: manifest), class: 'iiif-dnd pull-right', data: { turbolinks: false } do
```

Comment:
I _think_ we want the base of the link to go to an information page about IIIF on library.stanford.edu that was described in #679 (but maybe wasn't explicitly called out as acceptance criteria in the issue.

Reply:
Yes, we want a click to go to this page:

https://library.stanford.edu/projects/international-image-interoperability-framework/viewers

We might want to update that page but I think it is good enough to serve the purpose as is.

Reply:
Thanks for catching that. I've updated the PR to do that.